packages/cli/src/config.rs

```diff
@@ -0,0 +1,54 @@
+use anyhow::{Context, Result};
+use serde::{Deserialize, Serialize};
+use std::fs;
+use std::path::PathBuf;
+
+/// Global configuration for the CLI
+#[derive(Debug, Clone, Serialize, Deserialize, Default)]
+pub struct GlobalConfig {
+    /// The currently active space ID
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub active_space: Option<String>,
+}
+
+impl GlobalConfig {
+    /// Load global config from ~/.tonk/config.json
```

Comment:
Nit: I find it confusing to call this config because I'm thinking of active space more like active branch, so more of a state. It's also probably better to borrow from git here which store active branch info under `.git/HEAD` so maybe `.tonk/memory/space`.

I also think having atomic records as opposed to JSON might be a better (and probably why git does it) as it reduces changes of concurrent writes causing conflicts.